spec/models/comment_spec.rb

```diff
@@ -1,7 +1,31 @@
 require 'spec_helper'
 
 describe Comment do
-  it { is_expected.to validate_presence_of(:nickname) }
-  it { is_expected.to validate_presence_of(:comment) }
-  it { is_expected.to validate_length_of(:comment) }
+  let (:comment) { described_class.new }
+  describe 'Validations' do
+    context 'comment without nickname' do
+      it 'return false' do
+        comment.comment = 'Foo'
+
+        expect(comment).not_to be_valid
+      end
+    end
+
+    context 'comment without a comment' do
+      it 'return false' do
+        comment.nickname = 'Bar'
+
+        expect(comment).not_to be_valid
+      end
+    end
+
+    context 'comment with comment and nickname' do
+      it 'return true' do
+        comment.comment = 'Foo'
+        comment.nickname = 'Bar'
+
+        expect(comment).to be_valid
+      end
+    end
+  end
```

Comment:
it varies between people on the best strategy to test validations. some people prefer using `shoulda_matchers` that provides the `is_expected_to validate_presence_of(:foo)` syntax and others prefer using the predicate matcher with `valid?` using real examples.